src/fable-library-rust/src/DateTime.rs

```diff
@@ -0,0 +1,88 @@
+
+#[cfg(feature = "date")]
+pub mod DateTime_ {
+    use crate::{Native_::Lrc, String_::{string, self}};
+    use chrono::{DateTime as CDT, TimeZone, Utc, Local};
+
+    #[derive(Clone, Copy, PartialEq, PartialOrd, Debug)]
+    enum LocalUtcWrap {
+        CLocal(CDT<Local>),
+        CUtc(CDT<Utc>),
+    }
+
+    #[derive(Clone, Copy, PartialEq, PartialOrd, Debug)]
+    pub struct DateTime(LocalUtcWrap);
+
+    // impl core::fmt::Display for DateTime {
+    //     fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result {
+    //         write!(f, "{}", self.0.to_string())
+    //     }
+    // }
+
+    pub fn new_ymd(y: i32, m: i32, d: i32) -> DateTime {
+        let l = Local.ymd(y, m as u32, d as u32).and_hms(0, 0, 0);
+        DateTime(LocalUtcWrap::CLocal(l))
```

Comment:
Is this even correct? What does .NET do out of the box, local or UTC?

Reply:
@alexswan10k I think it's [unspecified](https://docs.microsoft.com/en-us/dotnet/api/system.datetime.-ctor?view=net-6.0#system-datetime-ctor(system-int32-system-int32-system-int32)).

Reply:
Hmm, that could be tricky to model. Will have to have a think. Perhaps NativeDate can be used as a 3rd option or something.

Reply:
@alexswan10k I agree it's confusing, I guess that's why the `DateTimeOffset` struct was introduced.
I guess we can consider switching the representation of `DateTime` to `chrono::naive::NaiveDateTime` + DateTimeKind.